README.md

```diff
@@ -126,7 +126,7 @@ The frontend part of Label Studio app lies in the `frontend/` folder and written
 
 ```
 cd label_studio/frontend/
-npm ci
+yarn install --frozen-lockfile
 npx webpack
```

Comment:
Should `npx webpack` also be removed? I tried running it after `yarn install` and recieved a long list of errors of this form:
```
ERROR in ./src/assets/icons/bulb.svg
Module build failed (from ./node_modules/@svgr/webpack/dist/index.js):
Error: Invalid plugins list. Provided 'plugins' in config should be an array.
```